### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by nimra200**
Add instructions on how to get the site running.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,26 @@
...
+Create a virtual environment
+
+``$ python3 -m venv virtualenv``
+
+``$ source virtualenv/bin/activate``
+
+Switch to flask directory and install requirements
+
+``$ cd flask``
+
```
Comment (calvh) [inline README.md:15]: I recommend having `pip install -r requirements.txt` instead of installing packages separately

**Review by calvh — CHANGES_REQUESTED** [review]
Thanks for doing this @nimra200 

I just have a minor comment about the requirements 

Also for next time, please try to prefix your branch name with a number (I recommend using the issue number) to keep branches unique and prevent conflicts (imagine if someone else also used the branch name "readme-installation")

so you can use something like 34-readme-installation

To get the number, just create a new issue.

**Comment by calvh** [comment]
@nimra200 Could you also please change the title from "Installation" to "Flask Development Installation"

I have found that setting the following environment variables in an `.env` file at the root of the `flask` folder (without adding python-dotenv package) works for me. If you could, please try replicating that and see if the app works on your end. If it does, then you can add that instruction to the list of steps.

```
FLASK_ENV
SECRET_KEY
MONGO_USER
MONGO_PASSWORD
DB
AUTH_SOURCE
HOSTNAME
PORT
```

@Rebecca-CYLi @HamdiaA appreciate if you could also confirm the above

**Review by Rebecca-CYLi — DISMISSED** [review]
good!

**Comment by calvh** [comment]
Might as well add tick the checkboxes to the requirements we have satisfied so far

**Comment by calvh** [comment]
Please see #105 and update accordingly

**Comment by calvh** [comment]
Add these to the top of the README right after the top-level header to include badges

License
Open Issues
Closed Issues
Open PRs
Closed PRs

Including this will fix #7

**Review by Rebecca-CYLi — APPROVED** [review]
wait, his latest review says "the readme is to describe the project". we need to change it to make sure the user knows what its doing

**Review by Rebecca-CYLi — CHANGES_REQUESTED** [review]
wait, his latest review says "the readme is to describe the project". we need to change it to make sure the user knows what its doing

**Comment by Rebecca-CYLi** [comment]
Ok, I added the description. New Review needed

**Comment by calvh** [comment]
Thanks @Rebecca-CYLi. The numbered lists don't display properly so I am fixing them now.

**Review by calvh — APPROVED** [review]

**Pull request merged**